index.js

```diff
@@ -29,44 +44,44 @@ try {
   const event_action = github.context.payload.action;
   console.log(`Event Action is: ${event_action}`);
 
+  let subject, body;
   // Build payload body
   if (github.context.eventName === 'issues') {
 
     if (event_action === 'opened') {
 
-      var subject = 'New Issue: ' + issue['title'];
-      var body = `${issue['user']['login']} created a  new issue. ${issue['body']}. More info here: ${issue['html_url']}`;
+      subject = 'New Issue: ' + issue['title'];
+      body = format_body(issue, issue['html_url'], 'opened');
 
       // output the payload to the console so the user can see it
       console.log(`Touchpoint subject is: ${subject}`);
       console.log(`Touchpoint body is: ${body}`);
 
-      create_touchpoint(subject, body)
+      create_touchpoint(subject, body);
 
     } else if (event_action === 'closed') {
 
-      var subject = 'Issue #: ' + issue['title'] + ' was closed';
-      var body = `${issue['user']['login']} closed an issue. ${issue['body']}. More info here: ${issue['html_url']}`;
+      subject = 'Issue #: ' + issue['title'] + ' was closed';
+      body = format_body(issue, issue['html_url'], 'closed');
 
     } else if (event_action === 'labeled') {
 
-      var subject = 'Issue #: ' + issue['title'] + ' was labeled';
-      var body = `${issue['user']['login']} labeled an issue. ${issue['body']}. More info here: ${issue['html_url']}`;
-      var label = github.context.payload.label;
+      subject = 'Issue #: ' + issue['title'] + ' was labeled';
+      body = format_body(issue, issue['html_url'], 'labeled');
```

Comment:
Tests are failing right now for task creation.We actually don't want to call format_body() for task creation because then it never matches the regex for the issue form.

Reply:
I believe I can solve this issue, while keeping the body formatted for totango. Thoughts?

Reply:
Actually I believe I just fixed it.

Reply:
Did you push it to this branch? I didn't see any changes in file.